site/content/docs/0.0/customize/color-modes.md

```diff
@@ -1,11 +1,364 @@
+{{< /callout >}}
+
+#### Change default behavior
+
+We use a custom Sass mixin, `color-mode()`, to help you control _how_ color modes are applied. By default, we use a `data` attribute approach, allowing you to create more user-friendly experiences where your visitors can choose to have an automatic dark mode or control their preference (like in our own docs here). This is also an easy and scalable way to add different themes and more custom color modes beyond light and dark.
+
+In case you want to use media queries and only make color modes automatic, you can change the mixin's default type via Sass variable. Consider the following snippet and its compiled CSS output.
+
+```scss
+$color-mode-type: data;
+
+@include color-mode(dark) {
+  .element {
+    color: var(--bs-primary-text-emphasis);
+    background-color: var(--bs-primary-bg-subtle);
+  }
+}
+```
+
+Outputs to:
+
+```css
+[data-bs-theme=dark] .element {
+  color: var(--bs-primary-text-emphasis);
+  background-color: var(--bs-primary-bg-subtle);
+}
+```
+
+And when setting to `media-query`:
+
+```scss
+$color-mode-type: media-query;
+
+@include color-mode(dark) {
+  .element {
+    color: var(--bs-primary-text-emphasis);
+    background-color: var(--bs-primary-bg-subtle);
+  }
+}
+```
+
+Outputs to:
+
+```css
+@media (prefers-color-scheme: dark) {
+  .element {
+    color: var(--bs-primary-text-emphasis);
+    background-color: var(--bs-primary-bg-subtle);
+  }
+}
+```
+
+#### Change the root selector
+
+You can also change the root selector from where the theme variables are set. By default, it's set to `:root`, but you can change it to any other selector. This is useful if you want to apply the theme to another element, for instance in Angular where you can't access easily the `<html>` element.
+
+```scss
+$ouds-root-selector: "#app";
+
+@include "@ouds/web";
+```
+
+Outputs to:
+
+```css
+#app,
+[data-bs-theme="light"],
+#app[data-bs-theme="light"] [data-bs-theme="root"],
+#app[data-bs-theme="dark"] [data-bs-theme="root-inverted"] {
+  /* Your light mode variables definition */
+}
+
+[data-bs-theme="dark"],
+#app[data-bs-theme="dark"] [data-bs-theme="root"],
+#app[data-bs-theme="light"] [data-bs-theme="root-inverted"] {
+  /* Your dark mode variables definition */
+}
+
+/* Further OUDS Web CSS */
+```
+
+<!-- OUDS mod: moved in `custom-libraries.md` the title ## Custom color modes -->
+
+## JavaScript
+
+To allow visitors or users to toggle color modes, you'll need to create a toggle element to control the `data-bs-theme` attribute on the root element, `<html>`. We've built a toggler in our documentation that initially defers to a user's current system color mode, but provides an option to override that and pick a specific color mode.
+
+{{< callout info >}}
+Check out our [Orange navbar mode selector variant]({{< docsref "/components/orange-navbar#with-mode-selector" >}}) to add a color mode selector to your navbar.
+{{< /callout >}}
```

Comment:
Maybe this callout should be hidden for now since the component is not available yet